### DISCUSSION TITLE
Explicit modeling of high-strain-rate deformation in DAMASK, such as in laser peening

### DISCUSSION THREAD

**Discussion by maziartour** [damask-multiphysics/DAMASK#20] [Q&A]
Hello, Is there any way to explicitly simulate plastic deformation with a high strain rate, such as in laser peening by DAMASK? I mean the same relations that are used for simulating high-strain-rate deformations by CPFE in ABAQUS/Explicit or by VUMAT subroutine in ABAQUS for crystal plasticity modeling.

**Comment by MarDiehl** [comment]
No, we don't support explicit solvers at the moment.
